.github/workflows/checkimage.yaml

```diff
@@ -0,0 +1,30 @@
+name: Regular base image update check
+on:
+  schedule:
+    - cron: "5 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Install skopeo
+        run: sudo apt-get install -y skopeo
+      - name: Check change
+        run: |
+          skopeo inspect "docker://$(grep -Po '(?<=FROM )([^"]+)' Dockerfile)" | grep -Po '(?<="Digest": ")([^"]+)' > .baseimagedigest
+          docker run --rm --entrypoint sh -u 0 quay.io/cloudservices/drift-backend:latest -c \
+            'yum upgrade -y --security > /dev/null; rpm -qa | sort | sha256sum' \
+            >> .baseimagedigest
+      - name: Do change if the digest changed
```

Comment:
for me to understand, which line checks here that the digest changed?

Reply:
```
git commit -m "chore(image): update and rebuild image" || echo "No new changes"
```
it will stop if there is no change to commit

Reply:
none :-)

or to be precise, it will not do a commit if it will not change at line https://github.com/RedHatInsights/drift-backend/pull/354/files#diff-06295df028ac079b768250e50f843f645028f83d2b38bbac57c49b64a6d95a14R26

Reply:
ah ok thanks :)